x-pack/plugins/asset_manager/docs/development.md

```diff
@@ -0,0 +1,34 @@
+# Asset Manager Plugin Development
+
+These docs contain information you might need if you are developing this plugin in Kibana. If you are interested in the APIs this plugin exposes, please see [./api.md](our API docs) instead.
+
+## Running Tests
+
+There are integration tests for the endpoints implemented thus far as well as for
+the sample data tests. There is also a small set of tests meant to ensure that the
+plugin is not doing anything without the proper config value in place to enable
+the plugin fully. For more on enabling the plugin, see [the docs page](./docs/index.md).
+
+The "not enabled" tests are run by default in CI. To run them manually, do the following:
+
+```shell
+$ node scripts/functional_tests_server --config x-pack/test/api_integration/apis/asset_manager/config_when_disabled.ts
+$ node scripts/functional_test_runner --config=x-pack/test/api_integration/apis/asset_manager/config_when_disabled.ts
+```
+
+The "enabled" tests are NOT run by CI yet, to prevent blocking Kibana development for a
+test failure in this alpha, tech preview plugin. They will be moved into the right place
+to make them run for CI before the plugin is enabled by default. To run them manually:
+
+```shell
+$ node scripts/functional_tests_server --config x-pack/test/api_integration/apis/asset_manager/config.ts
+$ node scripts/functional_test_runner --config=x-pack/test/api_integration/apis/asset_manager/config.ts
```

Comment:
The original `config.ts` test was split in two configurations, one for testing apis with assets source and the other for signals. Currently the [signals test suite is enabled](https://github.com/elastic/kibana/blob/main/.buildkite/ftr_configs.yml#L176-L177)

Reply:
Oh ok good call out, I'll take a look and maybe adjust slightly in my next PR